### ISSUE TITLE
Two tabs open on each debug - the debugger and the sam build

### ISSUE THREAD

**Issue by genachka** [aws/aws-toolkit-jetbrains#2848]
Recently upgraded from 1.29 to 1.32 and started noticing n both IntelliJ 2021.2.2 and PyCharm 2021.2.2 every time I debug a Lambda function, I see it opens two tabs where it used to have just one after the initial build is completed and the focus is left on the SAM build tab so it takes extra steps to close / switch to the debug tab.

Is this by design? And if yes, can there please be an option to have it work like before, where the sam build tab is automatically closed when it's ready for debugging.

![image](https://user-images.githubusercontent.com/7947294/135903167-41dedd4d-b25f-444d-8412-21e640a31c94.png)

**Comment by abrooksv** [comment]
Yes, this is by design. The old way prevented us from showing output if the debugger hadn't started such as SAM build failing in all use cases.

I can check if we can force the debugger tab to take focus upon opening though. I believe today it takes focus when a breakpoint gets hit.

**Comment by genachka** [comment]
Thanks for the quick response. I find it very counter productive. When debugging regular Java or Python I don't have to go searching for where the restart button / start debug button is -- which is now on the sam build tab, where as Java / Python direct is always on the debug tab. That along is very annoying. If at least it would be back on the debug tab I could live easier with having two tabs. Still not ideal from my point of view, but easier to swallow.
>Yes, this is by design. The old way prevented us from showing output if the debugger hadn't started in all use cases, such as SAM build failing.

It does. 
> I believe today it takes focus when a breakpoint gets hit.

**Comment by abrooksv** [comment]
> Also, I never had issues explicitly opening the build to see the error from sam in the past. 

It's an issue in only some of the IDEs (Rider for one), but the logic is the same for all of them so we had to go for the common denominator.

> When debugging regular Java or Python I don't have to go searching for where the restart button / start debug button is -- which is now on the sam build tab, where as Java / Python direct is always on the debug tab.

I know we tied the Stop button up. The lack of the Debug icon to trigger restart is a good call out. I can dig into the IntelliJ codebase and see how that is powered and if we can also wire that button in, but I can't guarantee we can make that work. We are limited by IntelliJ API

**Comment by abrooksv** [comment]
The button action is https://github.com/JetBrains/intellij-community/blob/master/platform/execution-impl/src/com/intellij/execution/runners/FakeRerunAction.java
